Resources/Prototypes/Entities/Structures/Machines/Computers/computers.yml

```diff
@@ -75,6 +75,20 @@
     color: "#c9c042"
   - type: Computer
     board: PowerComputerCircuitboard
+  - type: PowerMonitoringConsole
+    voltage: High
```

Comment:
this doesnt seem to be an actual field on this component?

Reply:
Fixed in ac07a2b5d932ee335d2eb99c6105a3eec0024cb6